src/java.base/share/classes/sun/nio/fs/AbstractFileSystemProvider.java

```diff
@@ -158,6 +158,7 @@ public boolean exists(Path file) {
     /**
      * Returns a path name as bytes for a Unix domain socket.
      * Different encodings may be used for these names on some platforms.
+     * If file is empty, then an empty byte[] is returned.
```

Comment:
This should be mis-read as "empty file" when you mean "empty path".

Reply:
Thanks. Have updated after last two comments.